saged/predict_tissue.py

```diff
@@ -156,7 +243,28 @@
     val_predictions = []
     val_true_labels = []
     val_encoders = []
+
+    tune_data = labeled_splits[0]
+
     for i in range(len(labeled_splits)):
+        # Select hyperparameters via nested CV
+        train_list = labeled_splits[:i] + labeled_splits[i+1:]
+
+        if not args.disable_optuna:
+            # optuna.logging.set_verbosity(optuna.logging.ERROR)
+            sampler = optuna.samplers.RandomSampler(seed=args.seed)
+            study = optuna.create_study()
+            print('Tuning hyperparameters...')
```

Comment:
How much longer does this typically take (compared to just running your experiments with fixed hyperparameters)?

Reply:
Roughly 10x as long. Running this took a week where it would normally take an afternoon or night